packages/location-management/src/ducks/types.ts

```diff
@@ -29,3 +29,8 @@ export interface APIEndpoints {
 
 /** Convenient type for either an array of jurisdictions or of tree nodes */
 export type TreeNodeType = OpenSRPJurisdiction | TreeNode;
+
+/** default component props */
+export interface Props {
+  opensrpBaseURL: string;
+}
```

Comment:
This does not feel like it belongs in ducks, maybe in a src/helpers/common file.